Dicom_Rust.ipynb

```diff
@@ -0,0 +1,526 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pydicom import dcmread\n",
+    "import pylibjpeg\n",
+    "import numpy as np\n",
+    "from tcia_utils import nbia\n",
+    "import pandas as pd\n",
+    "import requests"
```

Comment:
While PEP-8 only cares about sorting the inputs by "standard-library / third-party / own" and here all are third-party, I personally find it useful to start by the very popular ones, requests + numpy + pandas, and then the rest. Linters like to sort alphabetically, which I personally don't find too useful.

Also, my personal preference is to use `import tcia_utils` instead of `from tcia_utils import nbia`. When reading the code later, it's easier to remember than `tcia_utils` is a module and know where it's coming from, than having individual functions like `nbia`, which is not obvious if the function is defined in the notebook or imported.

Reply:
Agreed. I updated per your suggestion.